Library/OcFlexArrayLib/FlexArray.c

```diff
@@ -14,6 +14,19 @@
 
 #define INITIAL_NUM_ITEMS (8)
 
+VOID
+OcFlexArrayFreePointerItem (
+  IN  VOID  *Item
```

Comment:
Better have this be ``VOID **``, so that at least the caller needs a visible cast.

Reply:
Hang on, chiming in, since this is still how I wrote it.

My intention was that you should be able to pass this function in to OcFlexArrayInit without needing a cast, which seems natural to want to do? Why do you believe it *should* have a visible cast?

The more underlying logic is that items in FlexArray are just 'things', so the general method to free them most also just be a pointer to a thing. And then this useful utility method can't be passed in to the constructor, without needing a cast every time it is used, unless its parameter type is also 'pointer to thing'.

Reply:
It's not intended to be used directly, if that's relevant.

Reply:
In my opinion, do this outside of this PR.

Reply:
UPDATE: `OcFlexArrayFreePointerItem` should be left as is for the sake of polymorphism. No change shall be made. Resolved.